src/parquet/file/file-serialize-test.cc

```diff
@@ -41,13 +41,17 @@ class TestSerialize : public PrimitiveTypedTest<TestType> {
 
   void SetUp() {
     num_columns_ = 4;
+    num_rowgroups_ = 2;
+    rows_per_rowgroup_ = 50;
     this->SetUpSchema(Repetition::OPTIONAL, num_columns_);
   }
 
  protected:
   int num_columns_;
+  int num_rowgroups_;
+  int rows_per_rowgroup_;
 
-  void FileSerializeTest(Compression::type codec_type) {
+  void FileSerializeTest_(Compression::type codec_type, bool row_count_known) {
```

Comment:
reason for this rename?

Reply:
So FileSerializeTest is implemented further down, in terms of this helper function, keeping the same signature.